### PR TITLE
Inconsistent Text in PyTorch

### PR DESCRIPTION
**Pull request by goldmermaid**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1833/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1833/

**Review by AnirudhDagar — APPROVED** [review]
Thanks for the fix @goldmermaid :)

File: chapter_computer-vision/image-augmentation.md [hunk 1]
```diff
@@ -69,11 +69,18 @@ def apply(img, aug, num_rows=2, num_cols=4, scale=1.5):
 
 ### Flipping and Cropping
 
+:begin_tab:`mxnet`
 [**Flipping the image left and right**] usually does not change the category of the object. 
 This is one of the earliest and most widely used methods of image augmentation.
 Next, we use the `transforms` module to create the `RandomFlipLeftRight` instance, which flips
 an image left and right with a 50% chance.
```
Comment (AnirudhDagar) [inline chapter_computer-vision/image-augmentation.md:76]: I guess `:end_tab:` is missing here and in the following changes as well.
```suggestion
an image left and right with a 50% chance.
:end_tab:
```

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
Made a comment.

File: chapter_computer-vision/image-augmentation.md [hunk 2]
```diff
@@ -69,10 +69,18 @@ def apply(img, aug, num_rows=2, num_cols=4, scale=1.5):
...
+:end_tab:
+
+:begin_tab:`pytorch`
+[**Flipping the image left and right**] usually does not change the category of the object. 
+This is one of the earliest and most widely used methods of image augmentation.
+Next, we use the `transforms` module to create the `RandomHorizontalFlip` instance, which flips
+an image left and right with a 50% chance.
```
Comment (AnirudhDagar) [inline chapter_computer-vision/image-augmentation.md:83]:
```suggestion
an image left and right with a 50% chance.
:end_tab:
```

File: chapter_computer-vision/image-augmentation.md [hunk 3]
```diff
@@ -83,10 +91,16 @@ apply(img, gluon.data.vision.transforms.RandomFlipLeftRight())
 apply(img, torchvision.transforms.RandomHorizontalFlip())
 ```
 
+:begin_tab:`mxnet`
 [**Flipping up and down**] is not as common as flipping left and right. But at least for this example image, flipping up and down does not hinder recognition.
 Next, we create a `RandomFlipTopBottom` instance to flip
 an image up and down with a 50% chance.
```
Comment (AnirudhDagar) [inline chapter_computer-vision/image-augmentation.md:97]:
```suggestion
an image up and down with a 50% chance.
:end_tab:
```

File: chapter_computer-vision/image-augmentation.md [hunk 4]
```diff
@@ -83,10 +91,16 @@ apply(img, gluon.data.vision.transforms.RandomFlipLeftRight())
...
 
+:begin_tab:`pytorch`
+[**Flipping up and down**] is not as common as flipping left and right. But at least for this example image, flipping up and down does not hinder recognition.
+Next, we create a `RandomVerticalFlip` instance to flip
+an image up and down with a 50% chance.
```
Comment (AnirudhDagar) [inline chapter_computer-vision/image-augmentation.md:102]:
```suggestion
an image up and down with a 50% chance.
:end_tab:
```

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
@goldmermaid there are a couple of other places as well. Sorry for not mentioning those clearly in my last comment. :))

**Comment by mli** [comment]
Job d2l-en/PR-1833/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1833/

**Comment by mli** [comment]
Job d2l-en/PR-1833/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1833/

**Comment by mli** [comment]
Job d2l-en/PR-1833/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1833/

**Review by AnirudhDagar — APPROVED** [review]

**Pull request merged**